CHANGELOG.md

```diff
@@ -5,6 +5,10 @@ This CHANGELOG follows the format listed at [Keep A Changelog](http://keepachang
 
 ## [Unreleased]
 
+## [1.2.0] - 2017-03-16
```

Comment:
Please only add stuff under ` ## [Unreleased]` in PRs, the maintainers will decide on when to release.